conda-env/dev.yml

```diff
@@ -13,7 +13,7 @@ dependencies:
   - cartopy >=0.17.0
   - cartopy_offlinedata
   - cf-units
-  - dask
+  - dask <2024.12.0
```

Comment:
@chengzhuzhang @xylar FYI I was notified the performance issue will be fixed in `dask=2025.1.0`, https://github.com/dask/community/issues/410 via https://github.com/dask/dask/pull/11638 .

I think we should still constrain the dask version for now until 2025.1.0 is released and we do some more testing during the RC testing phase.

Reply:
Would it be reasonable to change this to `dask !=2024.12.0`?  I guess it's fine either way but that seems like we're we're likely to end up, right?

Reply:
I think either way would be fine. Before we are able to test 2025.1.0, I think `dask <2024.12.0` is safer for now.